src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -150,6 +179,105 @@ public static void main(final String[] args) {
     }
   }
 
+  public static FileDialog createFileDialog( ) {
+    return new FileDialog(mainFrame);
+  }
+
+  public static FileDialog newFileDialog() {
```

Comment:
Is there a semantic difference between `createFileDialog()` and `newFileDialog()`?